website/docs/faqs/run-one-model.md

```diff
@@ -2,11 +2,11 @@
 title: How do I run one model at a time?
 ---
 
-To run one model, use the `--models` flag (or `-m` flag), followed by the name of the model:
+To run one model, use the `--select` flag (or `-m` flag), followed by the name of the model:
```

Comment:
```suggestion
To run one model, use the `--select` flag (or `-s` flag), followed by the name of the model:
```

Reply:
might be understanding this wrong but do we want to also switch out the -m for -s to fit the new syntax?

Reply:
you're a prince among people for finding these, thank you @jasnonaz